numpy/core/code_generators/genapi.py

```diff
@@ -281,9 +281,10 @@ def find_functions(filename, tag='API'):
                     state = SCANNING
                 else:
                     function_args.append(line)
-        except Exception:
-            print(filename, lineno + 1)
+        except ParseError:
             raise
+        except Exception as e:
+            raise ParseError(filename, lineno + 1, "see chained exception for details") from e
```

Comment:
Lint is complaining that this line is too long, line lengths need to be < 80 characters.  This is a bit strange as there may be two ParseError exceptions.

Reply:
I see, I can shorten the line and push the changes.